.gitignore

```diff
@@ -1,6 +1,5 @@
-.idea/
```

Comment:
Why did you remove this one ?

Reply:
I copied the `.gitignore` from the Blog module.

It shouldn't even be there, in the first place though... Stuff like that should be in the individual developer's global `.gitignore`. See [here](https://thomashunter.name/blog/global-gitignore-vs-repository-gitignore/) for more information.

Reply:
> It shouldn't even be there, in the first place though... Stuff like that should be in the individual developer's global .gitignore. See here for more information.

I can't agree more, but it's quite difficult in the case of an open source project to force user do that. It's a security.

That being said, since v2 and the module subsplitting we've been building with Nicolas, nobody will open just the blog module and commit here so... yeah my comment useless i guess :)

Reply:
Technically Nick can force the contributor to do it himself. :D Just don't merge the PR until they do.

My only objection to adding it is that once you add one editor's metafiles to be ignored, it opens the gates for everyone else to want to add their editor's as well.

Reply:
I agree to remove it from the gitingore. No other repo has it, this is probably something from very early on.